### PR TITLE
First cut at a recipe for the RPFITS library.

### PR DESCRIPTION
**Pull request by pkgw**
There is absolutely no license information provided on [the official website](http://www.atnf.csiro.au/computing/software/rpfits.html) or in the source files. I will contact the author to try to get something specified.

### CODE REVIEW

**Comment by pkgw** [comment]
Blocked on the switch to conda-build 2.0, since these days conda-build 1.x seems to have a problem downloading FTP URLs.

**Comment by pkgw** [comment]
Please merge? I will update the deployed package (1) if/when I get any feedback from the maintainer about licensing and (2) when a new conda release is made [fixing the bug](https://github.com/conda/conda/pull/4276) so that it deals with the source package's FTP server correctly.

File: recipes/rpfits/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  detect_binary_files_with_prefix: true
+  skip: true  # [win]
+
+requirements:
+  build:
+    - gcc  # [osx]
+    - libgfortran  # [unix]
+    - toolchain
+  run:
+    - libgfortran  # [unix]
```
Comment (ocefpaf) [inline recipes/rpfits/meta.yaml]: If the package need `libquadmath` you will need `libgcc` on `linux`. (On `OS X` `libgfortran` does ship `libquadmath` consistency fun!)

File: recipes/rpfits/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  commands:
+    - rpfex </dev/null
+    - test -f $PREFIX/lib/librpfits.so  # [linux]
+    - test -f $PREFIX/lib/librpfits.dylib  # [osx]
```
Comment (ocefpaf) [inline recipes/rpfits/meta.yaml]: Now that we are using `conda-build 2` we can use:

```yaml
    - conda inspect linkages -p $PREFIX rpfits  # [not win]
    - conda inspect objects -p $PREFIX rpfits  # [osx]
```

I strongly encourage you to add that to package like this one. It helps a lot to debug linking issues.

File: recipes/rpfits/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+about:
+  home: http://www.atnf.csiro.au/computing/software/rpfits.html
+  license: unspecified
```
Comment (ocefpaf) [inline recipes/rpfits/meta.yaml]: Ouch!

Comment (pkgw) [inline recipes/rpfits/meta.yaml]: Yeah ... I've emailed the maintainer. One of the other big packages he's responsible for has good licensing practice, so hopefully ...

**Comment by pkgw** [comment]
Thanks!

**Pull request merged**